workflow/lib/sbs/align_cycles.py

```diff
@@ -27,6 +27,7 @@ def align_cycles(
     use_align_within_cycle=True,
     skip_cycles=None,
     manual_background_cycle=None,
+    manual_channel_mapping=None,
 ):
     """Rigid alignment of sequencing cycles and channels.
```

Comment:
This seems like a quite complex function. I wonder if we should include more documentation on these types of functions that is like "if X, we do Y, if a, we do be". Basically quite bullet point pseudo code about what is happening.